applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -14,51 +14,139 @@ Carrier event #XCARRIEREVT
 
 The ``#XCARRIEREVT`` is an unsolicited notification that indicates the event of the LwM2M carrier library.
 
-Unsolicited notification
-------------------------
-
-It indicates the event of the LwM2M carrier library.
-
```

Comment:
This heading getting removed has changed how Sphinx sees the heading levels, which is causing the build error. This is because heading level markers in Sphinx are relative, not absolute, so it's setting the Syntax heading for this entry one level higher now, and then complaining when it finds a different heading marker on Lines 156 and 407 where it's expecting the same level of heading.

Recommend restoring this part as the easy solution, even if it seems like redundant information.